scripts/release/release-test.R

```diff
@@ -922,6 +967,17 @@ verify_ne_model_class()
 verify_ne_imp_class()
 verify_ne_lht_class()
 
+logindata_1 <- create_dsi_builder(server = "testserver1", url = armadillo_url, profile = profile, password = admin_pwd, token = token, table = sprintf("%s/2_1-core-1_0/nonrep", project1))
+logindata_2 <- create_dsi_builder(server = "testserver2", url = armadillo_url, profile = profile, password = admin_pwd, token = token, table = sprintf("%s/2_1-core-1_0/nonrep", project1))
+logindata <- rbind(logindata_1, logindata_2)
```

Comment:
Maybe document this setup? Nice way to test 2 servers.

I guess you should use two different tokens to make it more real world

Reply:
I agree two tokens would be more real world, but I've kept it like this to try to keep test efficient and reduce number of tokens required.

Reply:
I think it has test value for ie 2 sessions breaking apart

Reply:
Ok done

Reply:
I've looked at this again, and whilst having two tokens is a nice suggestion we can't retrieve tokens in the CI (this was what caused it to fail). Therefore we would have to write two different test cases, one for the CI with the one token and one for running locally using two tokens combined with OIDC. So for now maybe we just keep the case with the one token that runs on both the CI and locally?

Reply:
CICD only runs test as admin as we cannot (yet?) connect to an OIDC account.

The section in the `README.md` could use some rewording.

```md
To prevent all the questions:

- copy the `dev.dist.env` to `.env`
- fill in some or all parts
- toggle the `interactive` to `n` or `y` to let the script wait for manual checks.
```

as it is not only useful  to prevent all questions.

Or we could extend documentation in the `dist.env`

Feel free to update that too in this PR

Reply:
Done